data/human/free worlds 0 prologue.txt

```diff
@@ -849,31 +926,32 @@ mission "FW Katya 3"
 			`	Eyes glances at her, sees her expression, and says, "No, Councilor, don't even think of it. That's a horrible idea."`
 			`	"No," she says, "it's brilliant. Captain <last>, are you familiar with the Dereliction and Salvage Act of 2976?"`
 			choice
-				`	"Of course. I read parliamentary proceedings every night when I'm trying to fall asleep."`
-					goto yes
 				`	"No, I've never heard of it."`
 					goto no
+				`	"Of course. I read parliamentary proceedings every night when I'm trying to fall asleep."`
 			
-			label yes
 			`	"Well then," she says, "you'll probably agree that legally speaking, Navy surveillance drones count as abandoned goods the moment their owners have left the system."`
 				goto response
 			
 			label no
 			`	"Well," she says, "it started when the Syndicate began experimenting with fully automated mining facilities on uninhabitable planets. Pirates started raiding the mines to steal equipment, and the Syndicate claimed that these installations were 'colonies' and it was the Navy's job to defend them. The Act was Parliament's response, stating that taking goods from a human being is piracy, but if the owner or some living representative of theirs is not present in the system, taking those goods is a legal salvage operation. The point was to keep the Navy from being obligated to defend every equipment cache or abandoned station in the galaxy. But by the same law, those Navy surveillance drones become abandoned goods the moment their motherships leave the system."`
-				goto response
 			
 			label response
 			choice
 				`	"Yup. That makes perfect sense to me."`
 					goto agree
+				`	"So first you have me smuggling people across the galaxy, and now you want me to attack Navy ships? Next you'll ask me to start taking prisoners."`
 				`	"That sounds like a terrible idea."`
-					goto disagree
+					goto terrible
+			
+			`	Eyes laughs, spitting out his milkshake in the process. "We don't be asking you to do anything of that sort," Katya responds somewhat aggressively. "Look, if we want to track down who dropped those bombs then we need to get our hands on better scanning equipment, equipment that the Navy has on their unmanned surveillance drones, and you're the one with a ship unconnected to the Free Worlds. All you need to do is find a lone surveillance drone to plunder. Legally the Navy can't to anything about it, and no one will be hurt in the process. If we succeed in finding who's responsible for the bombings, we could prevent war between the Republic and Free Worlds, saving thousands or even potentially millions of lives. So are you with us or not?"`
```

Comment:
Rather, it should be "even potentially, millions"

Reply:
I think the addition of commas would break the rhythm that I'm going for, as Tibe mentioned. It might not be 100% correct grammar, but this is character dialog, which we can be a little loose with.